docs/source/installation/osx.rst

```diff
@@ -61,9 +59,9 @@ but MacVim users need to install it using the following command::
 Fonts installation
 ==================
 
-Install downloaded patched font by double-clicking the font file in Finder, then 
-clicking :guilabel:`Install this font` in the preview window.
+Download the patched font from Github, install it by double-clicking the font file in Finder and then 
```

Comment:
It is actually not necessary that font was downloaded at all. Existing phrase is good except that “downloaded” does not have to be there.